CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## 0.21.1
+
+- fix: do not produce empty annotations
```

Comment:
As you did a little bit more than just that, could you add it also to the changelog pls? Thanks

Reply:
done